erpnext/manufacturing/doctype/job_card/job_card.py

```diff
@@ -550,10 +550,10 @@ def validate_sequence_id(self):
 					.format(message, bold(row.operation), bold(self.operation)), OperationSequenceError)
 
 	def validate_work_order(self):
-		if self.is_work_order_stopped():
-			frappe.throw(_("You can't make any changes to Job Card since Work Order is stopped."))
+		if self.is_work_order_closed():
```

Comment:
It should be both? This change will now allow working on job cards against stopped work order.